api-js/src/user/mutations/refresh-tokens.js

```diff
@@ -141,24 +145,31 @@ export const refreshTokens = new mutationWithClientMutationId({
       await trx.commit()
     } catch (err) {
       console.error(
-        `Trx commit error occurred while user: ${user._key} attempted to refresh tokens: ${err}`,
+        `Trx commit error occurred while user: ${userKey} attempted to refresh tokens: ${err}`,
       )
       throw new Error(i18n._(t`Unable to refresh tokens, please sign in.`))
     }
 
-    const newAuthToken = tokenize({ parameters: { userKey: user._key } })
+    const newAuthToken = tokenize({ parameters: { userKey } })
+
+    console.info(`User: ${userKey} successfully refreshed their tokens.`)
+
     const newRefreshToken = tokenize({
       parameters: { userKey: user._key, uuid: newRefreshId },
       expPeriod: 168,
       secret: String(REFRESH_KEY),
     })
 
-    console.info(`User: ${user._key} successfully refreshed their tokens.`)
+    response.cookie('refresh_token', newRefreshToken, {
+      maxAge: REFRESH_TOKEN_EXPIRY * 60 * 24 * 60 * 1000,
+      httpOnly: true,
+      secure: false,
+      sameSite: true,
```

Comment:
Is sameSite: true a valid value?

Reply:
sameSite: true is the same as sameSite: 'strict'. It used to binary but then they added 'lax' as an in-between